.env.sample

```diff
@@ -25,5 +25,10 @@ WP_TEST_MAILER_SMTP_PASSWORD=""
 WP_SVN_USERNAME=""
 WP_SVN_PASSWORD=""
 WP_TRANSIFEX_API_TOKEN=""
-WP_JIRA_USER="" // Jira username/email
-WP_JIRA_TOKEN="" // Jira token (https://confluence.atlassian.com/cloud/api-tokens-938839638.html) or password
+WP_JIRA_USER="" // JIRA username/email
```

Comment:
I would add a comment as in the premium plugin
```# Optional: for plugin deployment```

Reply:
It is a few lines above, actually.